### ISSUE TITLE
Unused property `hasDefaultValue`

### ISSUE THREAD

**Issue by denkv** [hobbit-project/ontology#17]
Looks like it's not used anywhere?

https://github.com/hobbit-project/ontology/blob/master/example/example.ttl#L51

**Comment by MichaelRoeder** [comment]
It is used in the front end: https://github.com/hobbit-project/platform/blob/master/hobbit-gui/gui-serverbackend/src/main/java/de/usu/research/hobbit/gui/rabbitmq/RdfModelHelper.java#L211

**Comment by denkv** [comment]
Yes, but that's `defaultValue`.

**Comment by MichaelRoeder** [comment]
You are right. Following the implementation at https://github.com/hobbit-project/core/blob/master/src/main/java/org/hobbit/vocab/HOBBIT.java `hasDefaultValue` is neither there nor in the `ontology.ttl` file :open_mouth: 

So I think that in the `example.ttl` file `hasDefaultValue` should be renamed to `defaultValue` instead of being removed.
